### ISSUE TITLE
Add StandardScaler

### ISSUE THREAD

**Issue by tkilias** [exasol/data-science-utils-python#28]
## Background

- The StandardScaler is besides the MinMaxScaler one of the most important normalization methods. Many models depend on it. https://scikit-learn.org/stable/modules/generated/sklearn.preprocessing.StandardScaler.html

## Acceptance Criteria
- StandardScaler as SQL ColumnPreprocessor
- Fixed StandardScaler for scikit learn which doesn't need to be fitted
